api/bots_api/bots_test_lib.py

```diff
@@ -91,18 +91,33 @@ def mock_http_conversation(self, test_name):
             os.path.abspath(__file__)), '..', 'bots', self.bot_name, 'fixtures'))
         http_request_path = os.path.join(base_path, '{}_request.json'.format(test_name))
         http_response_path = os.path.join(base_path, '{}_response.json'.format(test_name))
+        http_headers_path = os.path.join(base_path, '{}_headers.json'.format(test_name))
```

Comment:
Do we want a new file for every mock http test here (two is imo already a lot), or can we just integrate this bot's header info in a `meta` key in the `_request.json` file, like in the giphy bot test fixtures?